go.mod

```
@@ -6,7 +6,7 @@ require (
 	github.com/Masterminds/semver/v3 v3.2.1
 	github.com/aws/aws-sdk-go-v2/config v1.18.16
 	github.com/compose-spec/compose-go/v2 v2.0.0-rc.3
-	github.com/containerd/console v1.0.3
+	github.com/containerd/console v1.0.4-0.20230706203907-8f6c4e4faef5
```

Perhaps add a comment here pointing to the reason we have to use this version, e.g.;

```suggestion
	github.com/containerd/console v1.0.4-0.20230706203907-8f6c4e4faef5 // include https://github.com/containerd/console/pull/66, which is not yet in a release.
```